TezosChain.ts

```diff
@@ -355,39 +355,8 @@ export class TezosChain extends pulumi.ComponentResource {
         return obj;
       },
       {}
-    );
-    params.helmValues["tezos_k8s_images"] = pulumiTaggedImages;
-
-    if (params.getNumberOfFaucetAccounts() > 0) {
```

Comment:
I moved this block down to line 142 and included inside of it the creation of the faucet ingress.